src/models/validationSchema.js

```diff
@@ -17,12 +17,19 @@ const SignUpSchema = Joi.object().keys({
     .required()
     .messages(errorMessage('Username')),
   email: Joi.string().email().required().messages(errorMessage('Email')),
+  dob: Joi.date().messages(errorMessage('Date')),
+  gender: Joi.string().max(50).messages(errorMessage('Gender')),
+  currency: Joi.string().max(50).messages(errorMessage('Currency')),
+  placeofliving: Joi.string()
+    .max(225)
+    .messages(errorMessage('Place of living')),
+  lang: Joi.string().max(225).messages(errorMessage('Language')),
```

Comment:
remove the validation schema file from the models folder and find a better place to put it. I know you are not the one who put it here but you can fix it

Reply:
yes let me do it!